java-client/src/main/java/co/elastic/clients/json/JsonpMapper.java

```diff
@@ -51,6 +54,11 @@
      */
     <T> void serialize(T value, JsonGenerator generator);
 
+    /**
+     * Serialize an object to its Json String representation
+     */
+    <T> String serializeToString(T value);
```

Comment:
We can actually implement this method fully in the interface and remove other updates to `DelegatingJsonpMapper` and `JsonpMapperBase`:

```suggestion
    default <T> String serializeToString(T value) {
        StringWriter writer = new StringWriter();
        try (JsonGenerator generator = jsonProvider().createGenerator(writer)) {
            serialize(value, generator);
        };
        return writer.toString();
    }
```